test_result.txt

```diff
@@ -0,0 +1,19 @@
+{"timestamp":"2024-07-10T10:02:39.173458Z","level":"INFO","fields":{"msg":"3 Before alice fund"},"target":"exec"}
```

Comment:
Should this results file be committed?

Reply:
No remove them. They are created by the load and soak test for future test analysis. I think I'll disable them by default.